packages/libs/coreui/src/stories/containers/NoteBox.stories.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { NoteBox, Props } from '../../components/containers/NoteBox';
+import { Story, Meta } from '@storybook/react/types-6-0';
+import { UIThemeProvider } from '../../components/theming';
+import { mutedGreen, mutedMagenta } from '../../definitions/colors';
+
+export default {
+  title: 'Containers/NoteBox',
+  component: NoteBox,
+} as Meta;
+
+const Template: Story<Props> = function Template(props) {
+  return (
+    <UIThemeProvider
```

Comment:
Is there a reason to add the theme here? Or is it just to check that we get the correct colors regardless of theme?